src/shared/components/copyDownloadControls/CopyDownloadControls.tsx

```diff
@@ -185,9 +186,10 @@ export class CopyDownloadControls extends React.Component<IAsyncCopyDownloadCont
     {
         // async update of the copy text, if the copy text ends up to be different than the previous one,
         // we show a modal to force a second copy action.
-        setTimeout(this.initCopyProcess(), 1);
+        setTimeout(this.initCopyProcess, 1);
```

Comment:
@onursumer

Reply:
good catch, we should remove the `setTimeout` function, and just call `this.initCopyProcess()`.